wlutil/wlutil.py

```diff
@@ -108,6 +108,9 @@ def initialize():
     Is safe and fast to call every time you load the library."""
     log = logging.getLogger()
 
+    if not os.path.exists(mnt):
+        run(['mkdir', mnt])
```

Comment:
Are the intermediate path components of `mnt` guaranteed to already exist?  The `-p` flag is necessary otherwise.
You can also use [`os.mkdir()`](https://docs.python.org/3/library/os.html#os.mkdir) or [`os.makedirs()`](https://docs.python.org/3/library/os.html#os.makedirs)  here instead of an external command.

Reply:
fixed.